src/pages/404.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { Box, Button, Container, HStack, Text } from "@chakra-ui/react";
+
+function NotFound() {
+  return (
+    <Container
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      my="10"

```

Comment:
Don't use the shorthand syntax so write it as `marginY` instead of `my`. This is so it adheres to some uniformity across the source code & also helps in improved readability.